package.json

```diff
@@ -81,7 +81,7 @@
     "react-dom": "16.10.1",
     "react-router": "3.2.4",
     "redux-logger": "3.0.6",
-    "semantic-release": "15.13.24",
+    "semantic-release": "17.2.3",
```

Comment:
a lot of changes between the 2 versions : https://github.com/semantic-release/semantic-release/releases?page=6 + https://github.com/semantic-release/semantic-release/releases/tag/v16.0.0